settings.gradle

```diff
@@ -1,11 +1,13 @@
 rootProject.name = 'okhttp-parent'
 
 include ':mockwebserver'
-if (!properties.containsKey('android.injected.invoked.from.ide') && System.getenv(
+
+if (properties.containsKey('android.injected.invoked.from.ide') || System.getenv(
         'ANDROID_SDK_ROOT') != null) {
-  // Currently incompatible with Intellij
+  // Currently incompatible with Intellij, use with Android Studio and from CLI with explicit flag
```

Comment:
Tracking bug?

Reply:
It's not really a bug I think, just not setup or expected to be

<img width="864" alt="image" src="https://user-images.githubusercontent.com/231923/62822868-54d60c00-bb81-11e9-9d6f-7ad374ea9285.png">